docs/pages/setup/admin/daemon.mdx

```diff
@@ -143,15 +154,44 @@ To upgrade a host to a newer version of Teleport, you must:
 
 </Admonition>
 
-## Further reading
+
+## Understanding Teleport daemon command line options for unit file configuration
+
+The `teleport install systemd` command includes a number of optional flags that you can use to strictly define parameters of the generated unit file and set where that output will be written.
+The following table includes all command line options available with the `teleport install systemd` command, a brief description of what they do, and our general recommended defaults tested on **Ubuntu 20.04 LTS**:
+
+
+| Flag | Description | General Recommended Default |
+| --- | ----------- | ---- |
+| --fd-limit | The maximum number of open file descriptors, definted by `LimitNOFILE` in the unit file. |  **--fd-limit=8192** |
+| --env-file | The path to the `EnvironmentFile` defined by the system unit file, containing any potential variables to be used in the configuration. | **--envfile=/etc/default/teleport** |
+| --pid-file | The path to the pid file containing the process identification number. | **--pid-file=/run/teleport.pid** |
```

Comment:
When would someone want to change this?

Reply:
These are largely just to allow for increased customizability and shouldn't deviate from the default configuration often, but do have some potential benefits. The `--fd-limit` and the open file descriptors mentioned relate to memory. Since the open file descriptors have to be suspended in memory this limit sets the bounds for the process. Competing processes or higher expected memory usage by teleport could call for an increase of the --fd-limit. 

 The file paths to the --env-file and --pid-file are good defaults, but on certain distros or other unique configurations someone setting up teleport may find themselves needing to store those files within a different directory either because the path they're trying to use is not a good default, files are stored in a different location, permissions need to be set more finely for services given a certain environment or use case etc.... 
 
 I'm happy to readjust wording around the flags if we wanted or needed, especially if you anticipate the way i'm presenting this information may cause some friction, it's possible there may be context I'm missing.

Reply:
The `--fd-limit` is the one most likely to need changing. We even document on our scaling page that large clusters should bump it to 65535.

The other two I imagine will work fine with the defaults in almost all cases. We just exposed flags for these because relying on hard-coded paths when there are so many linux distros out there with subtle differences felt like a bad idea.